1-BasicSetUp.sh

```diff
@@ -67,15 +67,19 @@ if [[ -d $zsh_folder ]];then
 fi
 
 spatialPrint "Setting up Zsh + Zim now"
-execute sudo apt-get install zsh -y
+# Ubuntu 16.04 has older zsh in repository
+if [[ $(cat /etc/os-release | grep "VERSION_ID" | grep -o -E '[0-9][0-9]' | head -n 1) -le 16 ]]; then
+    zsh_build_script=$(wget https://gist.githubusercontent.com/rsnk96/87229bd910e01f2ee7c35f96d7cb2f6c/raw/f068812ebd711ed01ebc4c128c8624730ab0dc81/build-zsh.sh -O -)
+    # Install specific version of zsh
+    zsh_versioned_script=$(echo $zsh_build_script | sed '/cd zsh/a git checkout zsh-5.8')
```

Comment:
Doesn't seem to be a very good way to do it, and doesn't work right now
Should this done only for 16.04 or for 18.04 as well?

Reply:
Do you think it should be done ofr 18.04 too? I think the apt version of zsh should be stable, so that should be fine...

Reply:
Same here, the apt one should be okay for 18.04
The build is working now, so marking this one as resolved